### DISCUSSION TITLE
rename .config to something else

### DISCUSSION THREAD

**Discussion by bjo81** [juicedata/juicefs#4289] [General]
If e.g. a juicefs mountpoint is used as a homedirectory, this leads to some issues, as a lot of software stores its configuration in a .config-subdirectory. This it not possible when juicefs exposes a .config-file in its mountpoint.

**Comment by SandyXSD** [comment]
You can add the `--prefix-internal` option to the `juicefs mount` command, see: https://juicefs.com/docs/community/command_reference/#mount-fuse-options
